src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+function getRound(num: number): number {
+  return Math.floor(num * 100) / 100;
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: Figure['shape'] = 'triangle',
```

Comment:
so u can use it like this
```suggestion
    public shape: Shape = Shape.Triangle,
```